### PR TITLE
PR for #15: Familiarize yourself with new lab template

### PR DESCRIPTION
**Pull request by lucamlouzada**
**Closes**: Issue #15. Merges issue branch with main.

In this issue we noted some minor improvements to the new lab template.

**Changes in this branch:**

- In https://github.com/gentzkow/GentzkowLabTemplate/commit/214714de67b4ebcb3c75b8e763e8649fbf3b188e, I changed the example slides .tex file to use the [moloch ](https://github.com/jolars/moloch)theme instead of the metropolis theme.
- In https://github.com/gentzkow/GentzkowLabTemplate/commit/cfeb9c2440c01e4415a65a5f139b2a0a69750952, I changed the lib/shell/run_latex file to ensure the cleanup part is always run 
- In https://github.com/gentzkow/GentzkowLabTemplate/commit/6e793d89a5dcc6f3b1f75ab7ea1d2881cfaca8a0, I fixed the examples/latex/README.md file so the wildcard is properly used in the sample lines of code

**Deliverables:**
- [`examples/latex/my_project_slides.tex`](https://github.com/gentzkow/GentzkowLabTemplate/blob/6e793d89a5dcc6f3b1f75ab7ea1d2881cfaca8a0/examples/latex/my_project_slides.tex)
- [`examples/latex/README.md`](https://github.com/gentzkow/GentzkowLabTemplate/blob/6e793d89a5dcc6f3b1f75ab7ea1d2881cfaca8a0/examples/latex/README.md)
- [`lib/shell/run_latex.sh`](https://github.com/gentzkow/GentzkowLabTemplate/blob/6e793d89a5dcc6f3b1f75ab7ea1d2881cfaca8a0/lib/shell/run_latex.sh)

@ShiqiYang2022, can you please review? Thanks!

### CODE REVIEW

File: examples/latex/README.md
Comment (ShiqiYang2022) [inline examples/latex/README.md:60]:
```suggestion
    cp "${REPO_ROOT}/2_analysis/output/"*.jpg "${MAKE_SCRIPT_DIR}/input/"
```

File: examples/latex/README.md
Comment (ShiqiYang2022) [inline examples/latex/README.md:61]:
```suggestion
    cp "${REPO_ROOT}/2_analysis/output/"*.tex "${MAKE_SCRIPT_DIR}/input/"
```

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]
@lucamlouzada Thanks for the great work here!

I think there is some inconsistencies in using the quotes. Maybe Windows has higher tolerance and does't report an error. But Mac will report errors for the lines I commented.

Could you please double check whether those proposed edits makes sense to you, run the scripts from your Mac end, and confirm everything goes well? After that I am happy to sign off.

**Comment by lucamlouzada** [comment]
Thanks @ShiqiYang2022 , just confirmed your change ran in my Windows as well so I went ahead and merged. Will add the summary to the closed issue.

**Comment by ShiqiYang2022** [comment]
@lucamlouzada Thanks so much for closing this PR! Everything looks great to me.

I have two minor comments about our workflow, those are documented in our [predoc manual - workflow](https://docs.google.com/document/d/15urblzRR2XaymiXhvlEVfCJCwuMQQ3KaF-l4ol68dnA/edit#heading=h.a4mrwyx0enhg). 

* Let's wait for the reviewer to approve the pull request and then merge if the PR is not urgent.
* Let's remember to ["Squash Merge"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges) the pull request.

It's hard to always remember what's written in the manual at the beginning, so it would be a good experience learning by doing (like this PR!).

**Comment by ShiqiYang2022** [comment]
Another note: Now that the changes are brought to `master`, would it be convenient for you to update the branches accordingly? Thanks!

**Comment by lucamlouzada** [comment]
Thanks @ShiqiYang2022 , I'll pay attention to those points next time. I have updated the relevant branches in [09c1f574](https://github.com/gentzkow/GentzkowLabTemplate/commit/09c1f5749ec2bc554ca1194a9d61fe8911a1c50c) for branch 16, and also for branch [18](https://github.com/gentzkow/GentzkowLabTemplate/tree/18-improve-error ). Could you please quickly revise and provide feedback on whether this was done correctly? Thanks again for all your help.

**Comment by ShiqiYang2022** [comment]
@lucamlouzada All looks great to me, thanks!!

**Pull request merged**
